### PR TITLE
Implemented Display Leaderboard function

### LINKED ISSUE #2: Fill In Leaderboard
Write the code to display a leaderboard in the right hand section on the main page.  First write the function to dynamically build the table from a hard coded array of user/averages.  Follow up with another task to read the actual array from the TErdle server API.

### PR DESCRIPTION
**Pull request by tkline12**
Implemented Display Leaderboard with hard coded values

### CODE REVIEW

File: js/game.js [hunk 1]
```diff
@@ -506,6 +507,18 @@ function displayLeaderboard() {
     ]
 
 
+    const tbody = document.getElementById('leader-table-body');
+    leaderData.forEach((leader)=>{
+        const row = document.createElement('tr');
+        tbody.appendChild(row);
+        const userData = document.createElement('td');
+        const avgData = document.createElement('td');
```
Comment (tbeerbower) [inline js/game.js:515]: Why did you use this variable name?

File: js/game.js [hunk 2]
```diff
@@ -506,6 +507,18 @@ function displayLeaderboard() {
...
+        row.appendChild(userData);
+        row.appendChild(avgData);
+        userData.innerText = leader.user;
+        avgData.innerText = leader.avg;
```
Comment (mgillott) [inline js/game.js:519]: Matt's Comment

File: js/game.js (hunk 1 above)
Comment (tkline12) [reply js/game.js:515]: I was just following directions, I promise

Comment (tbeerbower) [reply js/game.js:515]: ok sounds good to me

**Review by tbeerbower — APPROVED** [review]
This looks good to me.

**Pull request merged**